xiPy/paho_mqtt_client.py

```diff
@@ -806,6 +806,14 @@ def loop(self, timeout=1.0, max_packets=1):
         self._out_packet_mutex.release()
         self._current_out_packet_mutex.release()
 
+        pending_bytes = 0
+
+        if self._ssl:
+            pending_bytes = self._ssl.pending()
+
+        if pending_bytes > 0:
+            timeout = 0.0
+
         # sockpairR is used to break out of select() before the timeout, on a
```

Comment:
I'm not saying to do so, just a question here:
If the whole block skipped in case of "pending_bytes > 0" instead of setting timeout to 0.0 that would do it as well, right?